### PR TITLE
New storage

### PR DESCRIPTION
**Pull request by stasBigunenko**
Remade storage - return []byte or slice of []byte.
Test left the same. Cann't create tests with tt comfortable.
All other issues taken into account.

### CODE REVIEW

File: pkg/storage/newStorage/storageDB.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	if _, ok := sdb.Data[id]; !ok {
+		return nil, nil
+	}
+
+	sdb.Data[id] = b
+
+	return b, nil
+}
+
+func (sdb *StorageDB) Delete(_ context.Context, id uuid.UUID) error {
```
Comment (DrIhor) [inline pkg/storage/newStorage/storageDB.go]: Return (bool, err) to show if delete.

Comment (DrIhor) [reply pkg/storage/newStorage/storageDB.go]: Or return err if not exist.

**Review by DrIhor — APPROVED** [review]

File: pkg/storage/newStorage/storageDB.go (hunk 1 above)
Comment (stasBigunenko) [reply pkg/storage/newStorage/storageDB.go]: done

File: pkg/storage/newStorage/storageDB.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	"context"
+	"encoding/json"
+	"github.com/google/uuid"
+	"github.com/stasBigunenko/monorepa/model"
+	"sync"
+)
+
+type StorageDB struct {
+	Data map[uuid.UUID][]byte
+	mu   sync.Mutex
```
Comment (romanilchyshyn) [inline pkg/storage/newStorage/storageDB.go:13]: Here is fine but just FYI. 
You can use `RWLock` with `RLock()` on read operations which is non-blocking for parallel reads.

**Review by romanilchyshyn — APPROVED** [review]
Here is fine but just FYI.
You can use `RWLock` with `RLock()` on read operations which is non-blocking for parallel reads.

**Pull request merged**